README.md

```diff
@@ -28,16 +28,16 @@ GET /songs/:songid
 POST /songs
 - Inserts a new song into the database
 
-POST /likes/:userid_:songid
+POST /likes?song=songid&user=userid
 - Creates a new like entry from a specific user for the specified song ID
 
-DELETE /likes/:userid_:songid
+DELETE /likes/song/:songid/user/:userid
```

Comment:
just curious, why do you switch between query params and ids in the url path?

Reply:
This actually appears to be out of date. I had made further changes that replaced the entirety of those urls with query parameters.

Reply:
I can make another push now.